### DISCUSSION TITLE
How to access kafka externally?

### DISCUSSION THREAD

**Discussion by rohankrao** [strimzi/strimzi-kafka-operator#7636] [Q&A]
My Kafka cluster is deployed on minikube with the docker driver.
I want to access the cluster using nodeport but with the IP address of the host machine where minikube is running.
My client is a different machine.
I can access it from the host machine.
Is there any way to access Kafka from another machine?

**Comment by scholzj** [comment]
Minikube is not really designed for this kind of thing. It is designed for local development. Locally, when you run Minikube with a VM driver, node ports should work. WIth other drives such s the Docker driver, ingress might work. But Minikube normally doesn't expose outside of the machine where it runs. So to expose it that way, you would need to do some additional networking setup I guess to forward the ports and addresses to the outside.
